### PR TITLE
Stop overriding onBackPressed so I can leave the damn application

### PR DESCRIPTION
**Pull request by VicV**
Seriously guys I can't hit back to leave the app, whats with that.

### CODE REVIEW

**Comment by divad12** [comment]
Thanks for fixing that. :) Not sure why that was there in the first place.

Currently travelling in Europe; will merge and deploy when I get back to Canada in about two weeks.

**Comment by VicV** [comment]
@divad12 bump

**Comment by divad12** [comment]
Sorry @VicV, I'm really really busy this week. Definitely planning to get this merged, though, along with https://github.com/UWFlow/flow-android/pull/45, and definitely before Fall 2014 starts.

Thanks for being patient. Haven't forgotten though!

**Comment by VicV** [comment]
@divad12 pls

File: src/com/uwflow/flow_android/MainFlowActivity.java [hunk 1]
```diff
@@ -288,14 +288,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
 
     @Override
-    public void onBackPressed() {
```
Comment (w2ji) [inline src/com/uwflow/flow_android/MainFlowActivity.java:291]: I think if you remove this, the back button will not go back to the previous fragment.

Comment (VicV) [inline src/com/uwflow/flow_android/MainFlowActivity.java:291]: This is not true at all. Test it out, don't speculate. Android automatically handles fragment transactions using the back button.

**Comment by KartikTalwar** [comment]
@divad12 bump

**Comment by VicV** [comment]
@divad12 I'll post a video of this all working today. If it doesn't, I'll fix it up.

**Comment by divad12** [comment]
Sorry for the delay on this. The original authors of this project have graduated, and I personally don't have time to maintain this project anymore. @VicV, if you have the bandwidth, I'd be happy if you took over as maintainer. If you'd like that, I'll send you deploy keys and instructions to your Gmail and get you set up to publish to the Play Store. I'll try to do a deploy this week if I have time to ensure it all works.

Thanks for submitting the pull request!

**Comment by VicV** [comment]
@divad12 the issue now is mainly that this project is incompatible with the newer versions of Android Studio / gradle. The build files are fucked up and in the wrong place so it cant actually be built any more. I'll look into this later, but yeah your shits broken yo

**Comment by divad12** [comment]
Hm. I don't believe any of us have tested this under Android Studio — we've always used Intellij. If you can get this working, let me know, and I can send you files so you can publish a signed APK.

**Pull request merged**